staging/src/k8s.io/kubectl/pkg/cmd/cmd_test.go

```diff
@@ -73,21 +74,21 @@ func TestKubectlSubcommandShadowPlugin(t *testing.T) {
 			expectPluginArgs: []string{"--bar", "--bar2", "--namespace", "test-namespace"},
 		},
 		{
-			name:        "test that a plugin executable is not found based on command args when also builtin subcommand does not exist",
-			args:        []string{"kubectl", "create", "foo2", "--bar", "--bar2", "--namespace", "test-namespace"},
-			expectError: "unable to find a plugin executable \"kubectl-create-foo2\"",
+			name:              "test that a plugin executable is not found based on command args when also builtin subcommand does not exist",
+			args:              []string{"kubectl", "create", "foo2", "--bar", "--bar2", "--namespace", "test-namespace"},
+			expectLookupError: "unable to find a plugin executable \"kubectl-create-foo2\"",
 		},
 		{
 			name:             "test that normal commands are able to be executed, when builtin subcommand exists",
-			args:             []string{"kubectl", "create", "role", "--bar", "--bar2", "--namespace", "test-namespace"},
+			args:             []string{"kubectl", "create", "job", "foo", "--image=busybox", "--dry-run=client", "--namespace", "test-namespace"},
 			expectPlugin:     "",
 			expectPluginArgs: []string{},
 		},
 		// rest of the tests are copied from TestKubectlCommandHandlesPlugins function,
 		// just to retest them also when feature is enabled.
 		{
 			name:             "test that normal commands are able to be executed, when no plugin overshadows them",
-			args:             []string{"kubectl", "get", "foo"},
+			args:             []string{"kubectl", "config", "get-clusters"},
```

Comment:
What is the reason of this change?

Reply:
these are changed to commands that require no cluster connection, so they can be tested without failing